package.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
let's nest this (and the lockfile) inside `tools/markdown-checker` - no need to clutter the main repo with it.